onnx_tf/handlers/frontend/size.py

```diff
@@ -0,0 +1,12 @@
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_op
+
+
+@onnx_op("Size")
+@tf_op("Size")
+class Size(FrontendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
```

Comment:
Size op is from 1 not 9. 
https://github.com/onnx/onnx/blob/master/docs/Operators.md#version-88